vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/frontend/src/grid-on-client-and-slot.js

```diff
@@ -17,13 +17,23 @@ class GridOnClientAndSlot extends LitElement {
 
   render() {
     return html`
-      <vaadin-grid id="tree">
-        <vaadin-grid-column>
-          <template>
-            <vaadin-grid-tree-toggle leaf="[[item.leaf]]" expanded="{{expanded}}" level="[[level]]">
-              <div class="tree-cell">[[item.name]]</div>
-            </vaadin-grid-tree-toggle>
-          </template>
```

Comment:
@web-padawan I needed to change a couple of tests that were still using the `<template>` API to use `renderer` instead